### ISSUE TITLE
failed to bench antidote

### ISSUE THREAD

**Issue by nherbaut** [goncalotomas/FMKe#191]
from f8a7a83cc5a6e70c41a15d9f2230ce8d3eb5a37f

I have the following results when trying to bench antidote (same for riak). Am I missing somethin?

```bash
nherbaut@nherbaut-Precision-5520:~/workspace/FMKe$ make select-antidote
./scripts/config/change_db.sh antidote
Changed target_database to antidote.
success
nherbaut@nherbaut-Precision-5520:~/workspace/FMKe$ make bench-antidote
rm -rf _build/default/rel
rm -rf _build/default/rel/
rebar3 release -n fmke
===> Verifying dependencies...
===> Plugin protobuffs does not export init/1. It will not be used.
===> Plugin protobuffs does not export init/1. It will not be used.
===> Compiling fmke
===> Starting relx build process ...
===> Resolving OTP Applications from directories:
          /home/nherbaut/workspace/FMKe/_build/default/lib
          /usr/local/lib/erlang/lib
===> Resolved fmke-0.1.0
===> Including Erts from /usr/local/lib/erlang
===> release successfully created!
./travis.sh bench normal antidote
> Setting up FMKe to benchmark antidote...
Changed target_database to antidote.
success
Changed database_ports to [8087].
make[1]: Entering directory '/home/nherbaut/workspace/FMKe'
rm -rf _build/default/rel
rm -rf _build/default/rel/
rebar3 release -n fmke
===> Verifying dependencies...
===> Plugin protobuffs does not export init/1. It will not be used.
===> Plugin protobuffs does not export init/1. It will not be used.
===> Compiling fmke
===> Starting relx build process ...
===> Resolving OTP Applications from directories:
          /home/nherbaut/workspace/FMKe/_build/default/lib
          /usr/local/lib/erlang/lib
===> Resolved fmke-0.1.0
===> Including Erts from /usr/local/lib/erlang
===> release successfully created!
make[1]: Leaving directory '/home/nherbaut/workspace/FMKe'
trying to load antidote docker image...
Using default tag: latest
latest: Pulling from mweber/antidotedb
Digest: sha256:e80e918e05525f79789ca78de48431eb7952e910c2d38b7e8d23036f5daeffd9
Status: Image is up to date for mweber/antidotedb:latest
antidote
antidote started.
compiling FMKe...
===> Verifying dependencies...
===> Plugin protobuffs does not export init/1. It will not be used.
===> Plugin protobuffs does not export init/1. It will not be used.
===> Compiling fmke
starting FMKe...
NODE_NAME: fmke@127.0.0.1
COOKIE: fmke
IP: 127.0.0.1
HTTP_PORT: 9090
waiting for FMKe to start...
FMKe started.
\033[32m> Populating antidote via FMKe using standard benchmark configuration...\033[0m\n
./scripts/populate_fmke.escript:177: syntax error before: 'case'
escript: There were compilation errors.
Makefile:17: recipe for target 'bench-antidote' failed
make: *** [bench-antidote] Error 127
```

**Comment by goncalotomas** [comment]
I left out the commit that fixed this in #188 inadvertently. Before merging #192 I'm going to add a minimal test suite that uses the population script so that this does not happen again.

Thank you so much for reporting this! 👍

**Comment by goncalotomas** [comment]
I have decided to leave in `populate_fmke.escript` as a convenience tool for running testing or trivial benchmarks. Since this issue was created I've written https://github.com/goncalotomas/fmke_populator , which is now the tool that should be used in order to populate the databases through FMKe.

Thanks again for taking the time for reporting the bug, and please let me know if for some reason things still don't work for you.
